fastlane_core/lib/fastlane_core/project.rb

```diff
@@ -342,10 +342,14 @@ def xcodebuild_parameters
         proj << "-clonedSourcePackagesDirPath #{options[:cloned_source_packages_path].shellescape}"
       end
 
-      if xcode_at_least_11 && options[:disable_package_automatic_updates]
+      if xcode_at_least_11 && options[:skip_package_dependencies_resolution]
         proj << "-disableAutomaticPackageResolution"
       end
 
+      if xcode_at_least_11 && options[:disable_package_automatic_updates]
```

Comment:
I'm so torn between the previous and the current implementation... and truth is, IMO, none of them are 100% accurate 😬 

From what I can tell, here's an analogy of what's happening:

- Parameter name: `im_sweet`
- Parameter description: `describes a salty parameter`
- Logic using the parameter: informs the cook that the dish should be salty.

What is leading to confusion is that the parameter name and its description are not a match. This PR aims to make changes so that the implementation is accurate to the parameter name, but this leaves the parameter description wrong, as @lacostej pointed out.

Another solution could be to simply change the parameter description, since the name and logic are a match, apparently. But doing just this wouldn't solve the issue since there's nothing preventing the package *_updates_* which was the initial motivator behind this change. 😬 

So I guess if we wanted to fix everything, we'd have to change the logic to match the parameter name (as OP is doing), as well as change the description. But then, are we sure the original author who implemented these parameters didn't really mean what they meant when they first implemented it? My point being like _"the documentation says the parameter is salty, shouldn't we keep it making recipes salty, even though its name indicates it should be sweet?"_

Let me know if any of this makes sense @bspinner @lacostej 😅 

Overall, I guess I'm ok with changing the logic and the presumptions made around the original author's intent.  But the least we should do is update the parameter description to be 100% clear of the new intent then, e.g. explicitly mention the parameters they translate to in `xcodebuild`, (`-skipPackageUpdates` and `-disableAutomaticPackageResolution`) as well as give a crystal clear definition of what it means, and the distinction between `package update` and `package resolution` 👀